tests/SConscript

```diff
@@ -2,4 +2,5 @@
 import os
 from lsst.sconsUtils import scripts, env
 
-scripts.BasicSConscript.tests()
+# testCoadds.py must be entirely skipped, and Scons doesn't like that
+scripts.BasicSConscript.tests(ignoreList=['testCoadds.py'])
```

Comment:
@timj with `pytest`:

    tests/testCoadds.py ssssssssssss

    ============================================================== 12 skipped in 1.52 seconds ===============================================================

with `scons tests`:

    running tests/testCoadds.py... failed
    Failed test output:
    tests/testCoadds.py

    [redacted]/lsstsw2/miniconda/lib/python2.7/site-packages/astropy/config/configuration.py:687: ConfigurationMissingWarning: Configuration defaults will be used due to OSError:Could not find unix home directory to search for astropy config dir on None
      warn(ConfigurationMissingWarning(msg))
    [redacted]/lsstsw2/miniconda/lib/python2.7/site-packages/matplotlib/font_manager.py:273: UserWarning: Matplotlib is building the font cache using fc-list. This may take a moment.
      warnings.warn('Matplotlib is building the font cache using fc-list. This may take a moment.')
    Traceback (most recent call last):
      File "tests/testCoadds.py", line 468, in <module>
        setup_module(sys.modules[__name__])
      File "[redacted]/lsstsw2/miniconda/lib/python2.7/unittest/case.py", line 62, in skip_wrapper
        raise SkipTest(reason)